<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>License Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOk">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContactName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompany">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContactNum">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLicenseNum">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>License Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNotes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>811</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
